src/content/docs/infrastructure/microsoft-azure-integrations/get-started/azure-native.mdx

```diff
@@ -5,47 +5,47 @@ metaDescription: 'Manage New Relic and Azure integrations with agents installed
 
 import infrastructureMonitorResources from 'images/monitored-resources.png'
 
-If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you deploy and manage New Relic directly through the Azure Marketplace. With it, you can: 
+If your team deploys software and cloud services through Azure, our [Azure Native New Relic Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/newrelicinc1635200720692.newrelic_liftr_payg) lets you manage your Azure-New Relic configuration directly from your Azure account. 
 
-* Create a free New Relic account or purchase New Relic directly from your Azure Portal
-* Manage billing directly through Azure Marketplace
-* Deploy New Relic APM and infrastructure agents through an automated setup flow 
-* Manage the metrics and logs for all your resources through an interactive dashboard
+The Azure Native New Relic Service provides the following capabilities:
 
-## Available features [#features]
+* Easily onboard and use New Relic as a natively integrated service on Azure.
+* Get a single bill for all the resources that you consume on Azure, including New Relic.
+* Automatically monitor Azure subscription activity and resource logs for New Relic.
+* Automatically monitor Azure metrics by using New Relic.
+* Use a single experience to install and uninstall the New Relic agent on virtual machines and app services.
 
-You can deploy the following agents through the Azure Native New Relic Service:
+On top of that, you can then use New Relic for other non-Azure-related monitoring you might need, using any of [our 500+ integrations](https://newrelic.com/instant-observability). 
 
-* The .NET and Java APM agents
-* The infrastructure agent, for Windows and Linux systems
+## Notes on available New Relic features [#features]
 
-Other agents and integrations are available. You can browse them from within New Relic or on our [Instant Observability page](https://newrelic.com/instant-observability).
+Some notes about the New Relic account created with this service: 
```

Comment:
It's that two-bullet list again...

Reply:
With "Notes" in the header, I don't think you need "notes" again in that intro sentence.